apps/dev-portal-web/domains/docs/widgets/AmountDistributionCalculator.tsx

```diff
@@ -0,0 +1,352 @@
+import { Col, Row, RowProps } from 'antd'
+import React, { CSSProperties, useCallback, useEffect, useState } from 'react'
+import { useIntl } from 'react-intl'
+
+import { split, createRecipientKey } from '@open-condo/billing/tools/billingCentrifuge'
+import type { TSplit, TDistributionItem, TSplitOptions } from '@open-condo/billing/tools/billingCentrifuge'
```

Comment:
What's the `T` comes from? We dont use this notation in condo project

Reply:
`T` means type
`I` means interface
Why not? We have no naming convention as far as I know.